requirements.txt

```diff
@@ -2,19 +2,20 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
 
+Flask==0.10.1
```

Comment:
got to ask... looks there might be a reason for this remaining `0.10.x` - do you know why?